requirements/test.txt

```diff
@@ -4,112 +4,428 @@
-psutil==5.8.0             # via -r requirements/base.txt, edx-django-utils
-py==1.10.0                # via pytest, tox
-pyasn1==0.4.8             # via python-jose, rsa
-pycparser==2.20           # via cffi
-pyparsing==2.4.7          # via -r requirements/base.txt, packaging
-pyrsistent==0.17.3        # via jsonschema
-pytest-cov==2.7.1         # via -c requirements/constraints.txt, -r requirements/test.in
-pytest-django==3.7.0      # via -c requirements/constraints.txt, -r requirements/test.in
-pytest==4.5.0             # via -c requirements/constraints.txt, -r requirements/test.in, pytest-cov, pytest-django
-python-dateutil==2.4.0    # via -c requirements/constraints.txt, -r requirements/base.txt, botocore, faker, freezegun, moto, xblock
-python-jose==3.2.0        # via moto
-python-slugify==4.0.1     # via -r requirements/base.txt, code-annotations
-python-swiftclient==3.11.0  # via -c requirements/constraints.txt, -r requirements/base.txt
-pytz==2021.1              # via -r requirements/base.txt, django, edx-submissions, fs, moto, xblock
-pyyaml==5.4.1             # via -r requirements/base.txt, cfn-lint, code-annotations, edx-i18n-tools, moto, xblock
-requests==2.25.1          # via -r requirements/base.txt, docker, moto, python-swiftclient, responses
-responses==0.12.1         # via moto
-rsa==4.7.1                # via python-jose
-s3transfer==0.3.4         # via -r requirements/base.txt, boto3
-six==1.15.0               # via -r requirements/base.txt, aws-sam-translator, bleach, cfn-lint, django-simple-history, docker, ecdsa, edx-i18n-tools, freezegun, fs, fs-s3fs, html5lib, jsonschema, junit-xml, libsass, mock, moto, pytest, python-dateutil, python-jose, python-swiftclient, responses, tox, virtualenv, websocket-client, xblock
-sqlparse==0.4.1           # via -r requirements/base.txt, django
-sshpubkeys==3.3.1         # via moto
-stevedore==3.3.0          # via -r requirements/base.txt, code-annotations, edx-django-utils
-testfixtures==6.17.1      # via -r requirements/test.in
-text-unidecode==1.3       # via -r requirements/base.txt, faker, python-slugify
-toml==0.10.2              # via tox
-tox==3.21.4               # via -r requirements/test.in
-urllib3==1.26.3           # via -r requirements/base.txt, botocore, requests, responses
-virtualenv==20.4.2        # via tox
-voluptuous==0.12.1        # via -c requirements/constraints.txt, -r requirements/base.txt
-wcwidth==0.2.5            # via pytest
-web-fragments==1.0.0      # via -r requirements/base.txt, xblock
-webencodings==0.5.1       # via -r requirements/base.txt, bleach, html5lib
-webob==1.8.6              # via -r requirements/base.txt, xblock
-websocket-client==0.57.0  # via docker
-werkzeug==1.0.1           # via moto
-wrapt==1.11.2             # via -c requirements/constraints.txt, aws-xray-sdk
-xblock-sdk==0.2.2         # via -r requirements/test.in
-xblock==1.4.0             # via -r requirements/base.txt
-xmltodict==0.12.0         # via moto
+appdirs==1.4.4
```

Comment:
Summary of changes:

aws-sam-translator==1.34.0 to aws-sam-translator==1.35.0
boto3==1.17.8 to boto3==1.17.28
botocore==1.20.8 to botocore==1.20.28
cfn-lint==0.45.0 to cfn-lint==0.47.2
coverage==5.4 to coverage==5.5
cryptography==3.4.5 to cryptography==3.4.6
defusedxml==0.6.0 to defusedxml==0.7.1
docker==4.4.2 to docker==4.4.4
edx-django-utils==3.13.0 to edx-django-utils==3.15.0
faker==6.1.1 to faker==6.6.1
jsonpatch==1.28 to jsonpatch==1.32
jsonpointer==2.0 to jsonpointer==2.1
newrelic==6.0.1.155 to newrelic==6.2.0.156
python-swiftclient==3.11.0 to python-swiftclient==3.11.1
rsa==4.7.1 to rsa==4.7.2
tox==3.21.4 to tox==3.23.0
urllib3==1.26.3 to urllib3==1.26.4
webob==1.8.6 to webob==1.8.7
websocket-client==0.57.0 to websocket-client==0.58.0